deploy.py

```diff
@@ -8,7 +8,9 @@
 def run():
     regions = []
     ec2 = boto3.client("ec2")
-    response = ec2.describe_regions()
+    response = ec2.describe_regions(
+        RegionNames=["us-east-1"]
```

Comment:
I think this is an accidental commit from testing? We want to get all regions when actually running this

Reply:
I forgot to double check deploy.py before push. let me update